frontend-react/package.json

```diff
@@ -186,6 +186,5 @@
         "minimist": "^1.2.6",
         "webpack": "^5.71.0"
     },
-    "resolutions": {
-    }
+    "resolutions": {}
```

Comment:
`"react-scripts": "^5.0.0",` should be `"react-scripts": "^5.0.1",` in the package.json file.

The way you've just changed the lock file means if anyone deletes and regenerates it from the package.json, then the version bumps will be lost.

Reply:
`^x.x.x` syntax installs latest minor/patch version. If someone were to delete/regenerate the lock file they would get the latest `5` version, which is `5.0.1` at the moment.